shared-test-resources/TypeResolverTestSuite/ByteCodeHierarchyTest/src/Class4.java

```diff
@@ -0,0 +1,20 @@
+import java.io.Serializable;
+
+public class Class4 extends Class2 implements Serializable, Cloneable{
+
+    @Override
+    public void method1(){
+        System.out.println("I'm method 1 in Class4");
+    }
+
+    @Override
+    public void method2(){
+        System.out.println("I'm method2 in Class4");
```

Comment:
I think adding println to the test case will slow the test down. is this needed?